glide.lock

```diff
@@ -23,9 +23,14 @@ imports:
 - name: github.com/m3db/m3cluster
   version: b596a86b06f37c9150ea352d6ec1549ed19f304b
   subpackages:
+  - client
```

Comment:
lol, i think it's pinning the rest because of the client subpackage (which we're using but hadn't pinned earlier)